examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -51,10 +51,13 @@ fi
 cd $ROOT_DIR
 rm -rf vcm_1.0
 
+echo "copying in the venv"
 cp -r /project/s1053/install/venv/vcm_1.0/ .
 git submodule update --init --recursive
+echo "install requirements"
 vcm_1.0/bin/python -m pip install external/fv3gfs-util/
 vcm_1.0/bin/python -m pip install .
+vcm_1.0/bin/python -m pip install gitpython
```

Comment:
should no longer be needed, see
https://github.com/VulcanClimateModeling/daint_venv/blob/5dcd1355a7b603b10b07278eb50e2642129668ed/install.sh#L35